hail/python/hailtop/utils/utils.py

```diff
@@ -559,7 +559,9 @@ def is_transient_error(e):
         # 503 service unavailable, 504 gateway timeout
         return True
     if isinstance(e, hailtop.httpx.ClientResponseError) and (
-            e.status == 403 and 'rateLimitExceeded' in e.body):
+            e.status == 403 and ('rateLimitExceeded' in e.body or
+                                 'QUOTA_EXCEEDED' in e.body or
+                                 re.match(e.body, "Quota.*exceeded"))):
```

Comment:
The regular expression match seems redundant at first blush. What was your motivation for including it?

Looking through the logs, it seems like the JSON structure for HTTP responses is:
```
{"error": {"errors": [{"code": "QUOTA_EXCEEDED", "message": "Quota 'NAME' exceeded.  Limit: ..."}, ...], ...}, ...}
```
https://console.cloud.google.com/logs/query;query=%22QUOTA_EXCEEDED%22;timeRange=2021-06-23T12:31:00.000Z%2F2021-06-24T15:31:00.000Z;cursorTimestamp=2021-06-24T14:31:26.565885Z?project=hail-vdc


I can't seem to find any occurrences of `"exceeded.  Limit"` without the accompanying `"QUOTA_EXCEEDED"`.
https://console.cloud.google.com/logs/query;query=%22exceeded.%20%20Limit%22%0A-%22QUOTA_EXCEEDED%22;timeRange=2021-06-17T12:31:00Z%2F2021-06-24T15:31:00Z;cursorTimestamp=2021-06-24T14:31:26.565885Z?project=hail-vdc

Reply:
To be absolutely sure we were catching it. I'll take out the regular expression and if there's still a problem, we can revisit.